example-data/turtle/sio-model/phenotyping.ttl

```diff
@@ -0,0 +1,36 @@
+@prefix : <http://purl.org/ejp-rd/cde/v020/example-rdf/> .
+@prefix obo: <http://purl.obolibrary.org/obo/> .
+@prefix sio: <http://semanticscience.org/resource/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+
+:identifier_ a sio:SIO_000115 ;
+    sio:SIO_000020 :phenotypicrole_ ;
+    sio:SIO_000300 "uid_000007"^^xsd:string .
+
+
+:person_ a sio:SIO_000498;
+    sio:SIO_000228 phenotypicrole_ ;
+    sio:SIO:000008 phenotypicattribute_ .
+
+
+:phenotypicrole_  a obo:OBI_0000093, a sio:SIO_000016;
+    rdfs:label "Phenotyping Patient"^^xsd:string;
+    sio:SIO_000356 :phenotypicprocess_ .
+
+
+:phenotypicprocess_ a sio:SIO_000006 ;
+    rdfs:label "comparative phenotypic assessment"^^xsd:string;
+    sio:SIO_000229 :phenotypicoutput_ .
+
+
+:phenotypicoutput_ a sio:SIO_000015, obo:NCIT_C102741 ;
+    rdfs:label "Cerebellar ataxia"^^xsd:string ;
+    sio:SIO_000669 "1992-01-01"^^xsd:date;
+    sio:SIO_000628 :phenotypicattribute_ .
+
+:phenotypicattribute_ a SIO:000614, sio:SIO_0100565, <https://hpo.jax.org/app/browse/term/HP:0001251> .
```

Comment:
HPO url here is wrong? can you use this uri? obo:HP_0001251

Reply:
Fixed